models/DataObject/Concrete.php

```diff
@@ -140,7 +140,14 @@ protected function update(bool $isUpdate = null, array $params = []): void
 
                 if (method_exists($this, $getter)) {
                     $value = $this->$getter();
+
                     $omitMandatoryCheck = $this->getOmitMandatoryCheck();
+                    // when adding a new object, skip check on mandatory fields with default value
+                    if (empty($value) && !$isUpdate && $fd->getMandatory()
```

Comment:
Is it necessary to check `$fd->getMandatory()` here? I mean if default value exists then `$omitMandatoryCheck` is true anyways.

Reply:
Thank you, sounds right, it is not necessary